src/App.php

```diff
@@ -73,7 +73,8 @@ protected function updateModel(Config $config, array $projectDefinition): void
 
         // Update model
         $result = $this->gdClient->getProjectModel()->updateProject($config->getProjectPid(), $projectModel);
-        $this->logger->info('Model updated', $result);
```

Comment:
Když se nezměnil model, tak se do `$result` dostal `false` a php na tom shořel.

Reply:
nebolo by lepsie v pripade ze sa model nezmenil to zalogovat nejak kontextovo? Teraz ma nenapada spravna hlaska ale nieco ako `Model is up to date` alebo `Model is up to date, nothing to update.` ci? Pretoze v pripade nezmeneho  modelu bude len `Model updated`.

Reply:
Jasně!

Reply:
upravil jsem